### ISSUE TITLE
I am getting error at "from transformers import AutoTokenizer, AutoConfig, LLaMAForCausalLM, LLaMATokenizer"

### ISSUE THREAD

**Issue by sungkim11** [22-hours/cabrita#6]
ImportError: cannot import name 'LLaMAForCausalLM' from 'transformers' (/usr/local/lib/python3.9/dist-packages/transformers/__init__.py)

**Comment by Whadup** [comment]
Yeah, they changed around which letters are capitalized, currently it's `LlamaForCausalLM`

**Comment by stefangrotz** [comment]
This took me a long time, thanks.

Currently its LlamaForCausalLM and LlamaTokenizer
